src/test/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifierIntegrationTest.java

```diff
@@ -36,25 +43,34 @@
  */
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({DisplayURLProvider.class, Office365ConnectorWebhookNotifier.class, Run.class})
-public class Office365ConnectorWebhookNotifierTestInt {
+public class Office365ConnectorWebhookNotifierIntegrationTest {
+
+    private static final String REQUESTS_DIRECTORY = "requests" + File.separatorChar;
 
     public static final String JOB_NAME = "myFirstJob";
+
+    private static final String CAUSE_DESCRIPTION = "Started by John";
     private static final int BUILD_NUMBER = 167;
     private static final long START_TIME = 1508617305000L;
 
     private AbstractBuild run;
     private HttpWorkerAnswer workerAnswer;
 
+    static {
+        TimeUtilsTest.setupTimeZoneAndLocale();
```

Comment:
need to do this because success.json has time zone which must be properly converted